usage/Frequent-Issues.md

```diff
@@ -1,11 +1,11 @@
 ## Run Configuration Bug 
-Due to IntelliJ version differences and usage on Windows vs. Mac OS, the run configurations show problems, sometimes. Change the *classpath or module* under `Edit configurations...` back to the Chat Overflow root module, and everything should be fine.
+Due to IntelliJ version differences as well as differences on platforms (e.g. Windows vs. Mac OS) the run configurations could show problems. If this error occurs, change the `classpath or module` setting under `Edit configurations...` back to the Chat Overflow root module. Now everything should be working fine again.
```

Comment:
this whole paragraph is confusing to read, not sure if that could be improved though

Reply:
I think I got it, this is the new version:

Because of the differences in IntelliJ versions the run configurations could be broken. If they are broken, change the `classpath or module` setting in `Edit configurations...` to ChatOverflow root module. Now everything should be working fine again.

I left out the platform differences, because that is part of the confusing part. The end user has a simple explaination for the problem, so that should be fine :D